src/classes/i18n.js

```diff
@@ -183,13 +215,19 @@ export default class i18n {
             this.languages.push(default_lang);
         }
 
-        return new Promise(resolve => {
+        await new Promise(resolve => {
```

Comment:
we do not need to return anything anymore?

Reply:
We don't have to. We don't need the result from this function. As per mdn documentation for [async functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function)
> Async functions always return a promise.

```js
n_ > async function foo () { console.log('Called foo'); }
n_ > foo().then(() => { console.log('Receiving promise chain'); });
Called foo
Promise {
  <pending>,
  [Symbol(async_id_symbol)]: 1332,
  [Symbol(trigger_async_id_symbol)]: 1330,
  [Symbol(destroyed)]: { destroyed: false }
}
n_ > Receiving promise chain
```